docs/docs/spark-configuration.md

```diff
@@ -145,6 +145,61 @@ Using those SQL commands requires adding Iceberg extensions to your Spark enviro
+| spark.sql.iceberg.vectorization.enabled                | Table default                                                  | Enables vectorized reads of data files                                       |
+| spark.sql.iceberg.parquet.reader-type                  | ICEBERG                                                        | Sets Parquet reader implementation (`ICEBERG`,`COMET`)                 |
+| spark.sql.iceberg.check-nullability                    | true                                                           | Whether to perform the nullability check during writes                 |
+| spark.sql.iceberg.check-ordering                       | true                                                           | Whether to check the order of fields during writes                     |
+| spark.sql.iceberg.planning.preserve-data-grouping      | false                                                          | Whether to preserve the existing grouping of data while planning splits |
+| spark.sql.iceberg.aggregate-push-down.enabled          | true                                                           | Enables pushdown of aggregate functions (MAX, MIN, COUNT)              |
+| spark.sql.iceberg.distribution-mode                    | See [Spark Writes](spark-writes.md#writing-distribution-modes) | Controls distribution strategy during writes                           |
+| spark.wap.id                                           | null                                                           | [Write-Audit-Publish](branching.md#audit-branch) snapshot staging ID   |
+| spark.wap.branch                                       | null                                                           | WAP branch name for snapshot commit                                    |
+| spark.sql.iceberg.compression-codec                    | Table default                                                  | Write compression codec (e.g., `zstd`, `snappy`)                       |
+| spark.sql.iceberg.compression-level                    | Table default                                                  | Compression level for Parquet/Avro                                     |
+| spark.sql.iceberg.compression-strategy                 | Table default                                                  | Compression strategy (for ORC)                                         |
+| spark.sql.iceberg.data-planning-mode                   | Table default                                                  | Override for data planning mode                                        |
+| spark.sql.iceberg.delete-planning-mode                 | Table default                                                  | Override for delete planning mode                                      |
+| spark.sql.iceberg.advisory-partition-size              | Table default                                                  | Advisory size (bytes) for data planning                                |
```

Comment:
Advisory Size (bytes) used for writing to the Table when Spark's Adaptive Query Execution is enabled. Used to size output files.